kit/inputs/src/UIMaterialTextView/UIMaterialTextViewFloating/hooks/useExpandingValue.ts

```diff
@@ -46,7 +46,10 @@ export function useExpandingValue(
 
     React.useEffect(() => {
         const toValue = getPosition(isExpanded);
-        const callback = isExpanded && toValue !== expandingPosition.value ? onExpand : undefined;
+        if (toValue === expandingPosition.value) {
```

Comment:
Please a comment what it's for :)
Like what would you like to see here after a year?